app/models/maintenance_stats/stats/github/release_stats.rb

```diff
@@ -24,7 +26,7 @@ def get_stats
         private
 
         def last_release_date
-          @results.data.repository.releases.nodes.first&.published_at 
+          @results&.map { |node| DateTime.parse(node.published_at) }.first&.strftime('%FT%TZ')
```

Comment:
The `strftime` is to keep the publish date in the same format as what it was previously.

Reply:
`.iso8601` is there if you'd rather use that than `strftime`

Reply:
`iso8601` was returning something slightly different than what GitHub is sending back. I'm trying to keep the formatting the same as the stats that are already in the database.